pkg/kubelet/rkt/rkt.go

```diff
@@ -2103,20 +2102,28 @@ func (r *Runtime) PortForward(pod *kubecontainer.Pod, port uint16, stream io.Rea
 		}
 		return fmt.Errorf("more than one running rkt pod for the kubernetes pod [%s]", strings.Join(podlist, ", "))
 	}
+	listPod := listResp.Pods[0]
 
 	socatPath, lookupErr := exec.LookPath("socat")
 	if lookupErr != nil {
 		return fmt.Errorf("unable to do port forwarding: socat not found.")
 	}
 
-	args := []string{"-t", fmt.Sprintf("%d", listResp.Pods[0].Pid), "-n", socatPath, "-", fmt.Sprintf("TCP4:localhost:%d", port)}
-
-	nsenterPath, lookupErr := exec.LookPath("nsenter")
-	if lookupErr != nil {
-		return fmt.Errorf("unable to do port forwarding: nsenter not found.")
+	var args []string
+	var fwCaller string
+	if strings.Contains(r.config.Stage1Image, "kvm") {
```

Comment:
This won't work in the case that the alpha "stage1 override" annotation is used.

Does the rkt api-service return what stage1 a pod is using? If it doesn't, it probably should.

I'd also slightly prefer checking for a more or less exact match, not just `string.Contains` for this.